server/controllers/offer-type/addOfferType.js

```diff
@@ -1,3 +1,14 @@
-module.exports = (req, res) => {
-  res.send(' add new offer type');
+const { addOfferType } = require('../../database/queries/offer-type/postOfferType.js');
+
+module.exports = (req, res, next) => {
+  const name = req.body;
+  console.log(name);
```

Comment:
:sob:  please remove this
and always check the linter console for errors 
and check the PR checkList